AppinfoJsonFileType.js

```diff
@@ -34,6 +35,42 @@ var AppinfoJsonFileType = function(project) {
     this.pseudo = this.API.newTranslationSet(project.getSourceLocale());
 };
 
+var defaultMappings = {
+    "**/appinfo.json": {
+        "template": "[dir]/[resourceDir]/[localeDir]/[filename]"
+    }
+}
+/**
+ * Return the mapping corresponding to this path.
+ * @param {String} pathName the path to check
+ * @returns {Object} the mapping object corresponding to the
+ * path or undefined if none of the mappings match
+ */
+AppinfoJsonFileType.prototype.getMappings = function(pathName) {
+    if (typeof(pathName) === "undefined") {
+        return undefined;
+    }
+    var mappings, match;
+    var jsonSettings= this.project.settings && this.project.settings.json;
```

Comment:
settings.json is used by the ilib-loctool-json plugin as well. Maybe this should have a different settings name?

Reply:
I changed the name to `jsonMap`